template/core/domain/src/main/java/io/bloco/core/domain/GetBooks.kt

```diff
@@ -0,0 +1,11 @@
+package io.bloco.core.domain
+
+import io.bloco.core.data.network.OpenLibraryHttpClient
+import io.bloco.core.data.network.OpenLibraryService
+import java.net.http.HttpClient
+import javax.inject.Inject
+
+class GetBooks @Inject constructor(
+    client: OpenLibraryService
```

Comment:
We'll eventually switch to a BookRepository, or just skip Repositories like we've been doing in other projects?

Reply:
Since we want to add the DataStore layer I think it makes sense to have the BookRepository that delegates between the service and the repo...